src/app/components/BibPage/BackLink.jsx

```diff
@@ -14,8 +14,7 @@ const BibPage = ({ searchURL, searchKeywords = '' }) => {
       className="nypl-back-link"
       to={`/search?${searchURL}`}
     >
-      <LeftArrowIcon />
```

Comment:
If we are removing this component from here, can you remove the import statement above.